release/ray_release/byod/byod_train_persistence_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# This script is used to build an extra layer on top of the base anyscale/ray image
+# to run the train_multinode_persistence test.
+
+set -exo pipefail
+
+pip3 install -U torch fsspec s3fs gcsfs pyarrow>=9.0.0 pytest
```

Comment:
I would recommend to use the managed library dependencies in https://www.notion.so/anyscale-hq/OSS-Release-Tests-9e0b126f7dd542aa80e4a6f936b8267f to install pip

post_command is used for other test setup that does not install libraries

Reply:
Or maybe add a TODO to move this to managed dependencies, according to https://www.notion.so/anyscale-hq/OSS-Release-Tests-9e0b126f7dd542aa80e4a6f936b8267f, and deal with it soon in the future

Reply:
Sounds good we will follow up on this!